### DISCUSSION TITLE
RedirectIfAuthenticated browser history problem

### DISCUSSION THREAD

**Discussion by aldozumaran** [livewire/livewire#7005] [1. Help]
I have a problem with the RedirectIfAuthenticated middleware using wire:navigate attribute, 
middleware redirect to the dashboard page but the url not change!

Steps
1. authenticate user
2. go to the login page with "guest" middleware

problem: redirect to the "dashboard page" with" /login"

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
